### PR TITLE
#38 broken assets fixed

### PR DESCRIPTION
**Pull request by microwavekonijn**
- fixed broken play and sfx assets
- added .idea directory to .gitignore (used by Webstorm)

Signed-off-by: Microwavekonijn

### CODE REVIEW

**Comment by microwavekonijn** [comment]
#38

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ docker/db
 !docker/db/.gitkeep
 config/development/*
 config/envConfig.js
+/.idea
```
Comment (JamesLongman) [inline .gitignore:8]: .git/info/exclude would be a better home for this if possible as it's more of a personal IDE less of a project file

File: src/assets/music/play-assets.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ const fun = {
     },
     11: {
         name: 'Danger zone',
-        link: 'https://www.youtube.com/watch?v=58QOBqAWNzE',
+        link: 'https://www.youtube.com/watch?v=siwpn14IE7E',
         volume: 0.04
```
Comment (JamesLongman) [inline src/assets/music/play-assets.js:71]: None of the volumes have been adjusted for any of these links, someone will either need to QA this and change the volume's for the links as required or we can complete the volume normalisation that will hopefully be implemented with #18

File: src/assets/sfx/sfx-assets.js [hunk 3]
```diff
@@ -21,12 +21,12 @@ If you're hosting local and not on youtube you better have a good reason, i.e. D
             options: {volume: 0.8},
             description: 'ADMIN ABUSE!'
         },
-        bad: {
-            source: 'youtube',
-            link: 'https://www.youtube.com/watch?v=scgCjxDByLM',
-            options: {volume: 0.4},
-            description: 'bad bad bad'
-        },
+        // bad: {
```
Comment (JamesLongman) [inline src/assets/sfx/sfx-assets.js:24]: No need to comment it out, feel free to just completely remove it

**Review by JamesLongman — CHANGES_REQUESTED** [review]
Nice first PR, changes are required but we'll be leaving this open in any case until we've completed our v0.1.0 basic infrastructure patch

**Pull request closed without merging**